framework/modules/duplicate.rst

```diff
@@ -0,0 +1,42 @@
+Duplicate
+=========
+
+Config
+------
+
+The way duplicate entities are identified can be configured in the entity ``Duplicate_config``. ``Duplicate_field``
+entities are grouped into ``Duplicate_field_group`` entities. For each ``Duplicate_field`` a field to check and the
```

Comment:
evtl noch erwähnen dass felder innerhalb einer gruppe AND verknüpft sind und die gruppen untereinander OR verknüpft